sw/airborne/modules/checks/preflight_checks.c

```diff
@@ -43,8 +43,13 @@
 #define PREFLIGHT_CHECK_INFO_TIMEOUT 5
 #endif
 
-static struct preflight_check_t *preflight_head = NULL;
+#ifdef PREFLIGHT_CHECK_BYPASS
+bool preflight_bypass = PREFLIGHT_CHECK_BYPASS;
+#else
 bool preflight_bypass = FALSE;
+#endif
```

Comment:
For coherence, can it be written like this ?
```suggestion
#ifndef PREFLIGHT_CHECK_BYPASS
#define PREFLIGHT_CHECK_BYPASS FALSE
#endif

bool preflight_bypass = PREFLIGHT_CHECK_BYPASS;
```